### ISSUE TITLE
Janus MCU server in Talk container cannot connect to `127.0.0.1:8188`

### ISSUE THREAD

**Issue by STBoyden** [nextcloud/all-in-one#1220]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Install AIO
2. Enable Talk container
3. Open AIO
4. Observe Talk container logs

### Expected behavior 
The Janus MCU server should be able to connect to the 8188 port.

### Actual behavior 
Repeated: `main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s`, possibly being the cause of experienced issues on Nextcloud Talk.

### Host OS 
Rocky Linux 9 running Linux Kernel version 5.14.0.

#### Nextcloud AIO version 
2.0.3 (latest as of 2022-10-03).

#### Current channel 
`latest`

#### Other valuable info 

AIO is behind a NGINX reverse proxy and the domain is proxied via Cloudflare.

Shouldn't conflict but no services on the host are using port `8188` regardless.

SELinux is disabled and `3478/tcp` and `3478/udp` are open on the firewall.

See the following logs from the Talk container:

```
+ sed -i 's|#turn_rest_api_key.*|turn_rest_api_key = "84d2dd0bc699c82e874ce70cab606d4abf1ff3f86a7c9886"|' /etc/janus/janus.jcfg
+ sed -i 's|#full_trickle.*|full_trickle = true|g' /etc/janus/janus.jcfg
+ sed -i 's|#stun_server.*|stun_server = "127.0.0.1"|g' /etc/janus/janus.jcfg
+ sed -i 's|#stun_port.*|stun_port = 3478|g' /etc/janus/janus.jcfg
+ sed -i 's|#turn_port.*|turn_port = 3478|g' /etc/janus/janus.jcfg
+ sed -i 's|#turn_server.*|turn_server = "127.0.0.1"|g' /etc/janus/janus.jcfg
+ sed -i 's|#turn_type .*|turn_type = "udp"|g' /etc/janus/janus.jcfg
+ sed -i 's|#ice_ignore_list .*|ice_ignore_list = "udp"|g' /etc/janus/janus.jcfg
+ sed -i 's|#interface.*|interface = "lo"|g' /etc/janus/janus.transport.websockets.jcfg
+ sed -i 's|#ws_interface.*|ws_interface = "lo"|g' /etc/janus/janus.transport.websockets.jcfg
+ set +x
Janus commit: not-a-git-repo
Compiled on:  Wed May  5 14:28:53 UTC 2021

Running Janus as a daemon
Adding 1 external loggers
Logger plugins folder: /usr/lib/x86_64-linux-gnu/janus/loggers
Loading logger plugin 'libjanus_jsonlog.so'...
[WARN] JSON logger disabled
---------------------------------------------------
  Starting Meetecho Janus (WebRTC Server) v0.11.2
---------------------------------------------------

Checking command line arguments...
Debug/log level is 4
Debug/log timestamps are disabled
Debug/log colors are disabled
Adding 'vmnet' to the ICE ignore list...
Using 172.17.0.3 as local IP...
Token based authentication disabled
Initializing recorder code
Initializing ICE stuff (Full mode, ICE-TCP candidates disabled, full-trickle, IPv6 support disabled)
STUN server to use: 127.0.0.1:3478
  >> 127.0.0.1:3478 (IPv4)
Testing STUN server: message is of 20 bytes
[25] 2022/10/03 21:56:42.662478 [INF] Starting nats-server
[25] 2022/10/03 21:56:42.662524 [INF]   Version:  2.9.0
[25] 2022/10/03 21:56:42.662528 [INF]   Git:      [517d9b3]
[25] 2022/10/03 21:56:42.662530 [INF]   Name:     NCK7NCXVY7KLBDWW6SBIVBFF5RF3SN66BJU4PCBBMMU7UU4VHGCUXTYZ
[25] 2022/10/03 21:56:42.662533 [INF]   ID:       NCK7NCXVY7KLBDWW6SBIVBFF5RF3SN66BJU4PCBBMMU7UU4VHGCUXTYZ
[25] 2022/10/03 21:56:42.662538 [INF] Using configuration file: /etc/nats/nats.conf
[25] 2022/10/03 21:56:42.662746 [INF] Listening for client connections on 127.0.0.1:4222
[25] 2022/10/03 21:56:42.662751 [INF] Server is ready
main.go:133: Starting up version 33dc5a554b496bd253bedbecc6755de514a64867/go1.19 as pid 26
main.go:142: Using a maximum of 4 CPUs
natsclient.go:108: Connection established to nats://127.0.0.1:4222 (NCK7NCXVY7KLBDWW6SBIVBFF5RF3SN66BJU4PCBBMMU7UU4VHGCUXTYZ)
grpc_common.go:167: WARNING: No GRPC server certificate and/or key configured, running unencrypted
grpc_common.go:169: WARNING: No GRPC CA configured, expecting unencrypted connections
backend_storage_static.go:72: Backend backend-1 added for https://<redacted>.com/
hub.go:201: Using a maximum of 8 concurrent backend connections per host
hub.go:208: Using a timeout of 10s for backend connections
hub.go:304: Not using GeoIP database
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 1s
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
socket: Protocol not supported
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 2s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 4s
[FATAL] [ice.c:janus_ice_test_stun_server:1009] No response to our STUN BINDING test
[FATAL] [janus.c:main:4823] Invalid STUN address 127.0.0.1:3478
Error launching Janus (error code 1), check the logs for more details
Janus commit: not-a-git-repo
Compiled on:  Wed May  5 14:28:53 UTC 2021

Running Janus as a daemon
Logger plugins folder: /usr/lib/x86_64-linux-gnu/janus/loggers
Loading logger plugin 'libjanus_jsonlog.so'...
[WARN] JSON logger disabled
Adding 1 external loggers
---------------------------------------------------
  Starting Meetecho Janus (WebRTC Server) v0.11.2
---------------------------------------------------

Checking command line arguments...
Debug/log level is 4
Debug/log timestamps are disabled
Debug/log colors are disabled
Adding 'vmnet' to the ICE ignore list...
Using 172.17.0.3 as local IP...
Token based authentication disabled
Initializing recorder code
Initializing ICE stuff (Full mode, ICE-TCP candidates disabled, full-trickle, IPv6 support disabled)
STUN server to use: 127.0.0.1:3478
  >> 127.0.0.1:3478 (IPv4)
Testing STUN server: message is of 20 bytes
  >> Our public address is 127.0.0.1
TURN server to use: 127.0.0.1:3478 (udp)
TURN REST API backend: (disabled)
Crypto: OpenSSL >= 1.1.0
No cert/key specified, autogenerating some...
Fingerprint of our certificate: C9:CA:3F:49:2A:DC:D2:91:90:53:6E:D1:66:9E:44:12:AA:59:F1:6A:03:49:4F:79:8F:A7:9F:45:ED:2D:6D:F2
Event handlers support disabled
Plugins folder: /usr/lib/x86_64-linux-gnu/janus/plugins
Sessions watchdog started
Joining Janus requests handler thread
Loading plugin 'libjanus_audiobridge.so'...
JANUS AudioBridge plugin initialized!
Loading plugin 'libjanus_duktape.so'...
[[36mechotest.js[0m]  Loading script...
[[36mechotest.js[0m]  Modules folder: /usr/share/janus/duktape
[[36mechotest.js[0m]  Loading module: janus-sdp
[[36mechotest.js[0m]  Module loaded
[[36mechotest.js[0m]  Script loaded
[[36mechotest.js[0m]  Initializing...
[[36mechotest.js[0m]  Initialized
Janus JavaScript plugin (Duktape) initialized!
Loading plugin 'libjanus_echotest.so'...
JANUS EchoTest plugin initialized!
Loading plugin 'libjanus_lua.so'...
[0m[[34mechotest.lua[0m][0m Loading...
[0m[[34mechotest.lua[0m][0m Loaded
[0m[[34mechotest.lua[0m][0m Initializing...
[0m[[34mechotest.lua[0m][0m Initialized
Janus Lua plugin initialized!
Loading plugin 'libjanus_nosip.so'...
JANUS NoSIP plugin initialized!
Loading plugin 'libjanus_recordplay.so'...
JANUS Record&Play plugin initialized!
Loading plugin 'libjanus_sip.so'...
JANUS SIP plugin initialized!
Loading plugin 'libjanus_streaming.so'...
[ERR] [plugins/janus_streaming.c:janus_streaming_init:2005] Can't add 'live' mountpoint, no such file '/usr/share/janus/streams/radio.alaw'...
[ERR] [plugins/janus_streaming.c:janus_streaming_init:2072] Can't add 'ondemand' mountpoint, no such file '/usr/share/janus/streams/music.mulaw'...
JANUS Streaming plugin initialized!
Loading plugin 'libjanus_textroom.so'...
JANUS TextRoom plugin initialized!
Loading plugin 'libjanus_videocall.so'...
JANUS VideoCall plugin initialized!
Loading plugin 'libjanus_videoroom.so'...
JANUS VideoRoom plugin initialized!
Loading plugin 'libjanus_voicemail.so'...
JANUS VoiceMail plugin initialized!
Transport plugins folder: /usr/lib/x86_64-linux-gnu/janus/transports
Loading transport plugin 'libjanus_http.so'...
HTTP transport timer started
HTTP webserver started (port 8088, /janus path listener)...
JANUS REST (HTTP/HTTPS) transport plugin initialized!
Loading transport plugin 'libjanus_mqtt.so'...
[ERR] [config.c:janus_config_parse:205] Error parsing config file at line 30: syntax error
[WARN] Couldn't find .jcfg configuration file (janus.transport.mqtt), trying .cfg
[ERR] [config.c:janus_config_parse:191]   -- Error reading configuration file 'janus.transport.mqtt.cfg'... error 2 (No such file or directory)
MQTT SSL support disabled
[WARN] MQTT support disabled for both Janus and Admin API, giving up
JANUS MQTT transport plugin destroyed!
[WARN] The 'janus.transport.mqtt' plugin could not be initialized
Loading transport plugin 'libjanus_nanomsg.so'...
JANUS Nanomsg transport plugin initialized!
Nanomsg thread started
Loading transport plugin 'libjanus_pfunix.so'...
[WARN] No Unix Sockets server started, giving up...
[WARN] The 'janus.transport.pfunix' plugin could not be initialized
Loading transport plugin 'libjanus_rabbitmq.so'...
RabbitMQ SSL support disabled
[WARN] RabbitMQ support disabled for both Janus and Admin API, giving up
[WARN] The 'janus.transport.rabbitmq' plugin could not be initialized
Loading transport plugin 'libjanus_websockets.so'...
libwebsockets logging: 0
[ERR] [transports/janus_websockets.c:janus_websockets_init:600] Error creating libwebsockets context...
[WARN] The 'janus.transport.websockets' plugin could not be initialized
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 8s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
main.go:228: Could not initialize janus MCU (dial tcp 127.0.0.1:8188: connect: connection refused) will retry in 16s
```

**Comment by szaimen** [comment]
This is an incorrext log entry and will be fixed with https://github.com/nextcloud/all-in-one/pull/1203

**Comment by STBoyden** [comment]
> This is an incorrext log entry and will be fixed with #1203

Ah apologies.

**Comment by szaimen** [comment]
This is now released with v2.0.4 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
